server/lists/adapters/__tests__/users.spec.js

```diff
@@ -21,4 +21,24 @@ describe('User Adapter', () => {
         expect(results).toBeDefined();
         expect(results).toMatchSnapshot();
     });
+
+    it('should correctly transform null values', async () => {
+        const mockData = [
+            { firstName: null, lastName: 'A', email: 'user.a@example.com', id: 1 },
+            { firstName: 'User', lastName: null, email: 'user.b@example.com', id: 2 },
+            { username: 'userC', firstName: 'User', lastName: 'C', email: null, id: 3 },
+            { username: 'userD', firstName: null, lastName: null, email: 'user.D@example.com', id: 4 },
+            { username: 'userE', firstName: null, lastName: null, email: null, id: 5 }
```

Comment:
I think there are maybe some other possible scenarios here:
- User has a username, firstname, email, but lastname null 
- User has a username, lastname, email , but firstname null
- User has a username, firstname, but lastname and email null
- User has a username, lastname, but firstname and email null
- User has a first name or last name of "null" eg "Jane Null"

Reply:
Done, I've added a username to most of the data and covered some of the missing potential states.